test/test_sdist_package.py

```diff
@@ -0,0 +1,29 @@
+import shutil
+from pathlib import Path
+
+from . import utils
+
+
+def test(capfd, tmp_path):
+    sdist_filename = "spam-0.1.0.tar.gz"
+    sdist = Path(__file__).parent / sdist_filename
+    project_dir = tmp_path / "project"
+    project_dir.mkdir(parents=True, exist_ok=True)
+
+    # this is a file, not a dir. But that's the variable name used
+    package_dir = project_dir / sdist_filename
+    shutil.copy2(sdist, package_dir)
+
+    # build the wheels from an sdist
+    actual_wheels = utils.cibuildwheel_run(
+        project_dir,
+        package_dir=package_dir,
+        add_env={
+            # this shouldn't depend on the version of python, so build only CPython 3.6
+            "CIBW_BUILD": "cp36-*",
```

Comment:
I'd probably go with something a bit newer, just so we don't have to mess with this when 3.6 goes away (mid next year, I think, but still). 3.9 or something.

I'd also really like to see that `--config-file=cibuildwheel.toml` still works with this (I _think_ so?), maybe this could be dropped into a toml file instead, and the equivalent of this passed in?

Reply:
> I'd probably go with something a bit newer, just so we don't have to mess with this when 3.6 goes away (mid next year, I think, but still). 3.9 or something.

I modeled this after the other tests for the pure wheel and it is using a 3.6 version. Therefore I thought that was the right way to process this and there must have been a reason for sticking to 3.6

Reply:
That was probably shiny and new at the time. 🤣 We can update them all at the same time, then, fine as is.

Reply:
> That was probably shiny and new at the time. rofl We can update them all at the same time, then, fine as is.

This was the intent in adopting the existing ways

Reply:
> I'd also really like to see that --config-file=cibuildwheel.toml still works with this (I think so?), maybe this could be dropped into a toml file instead, and the equivalent of this passed in?

can you elaborate what you mean a bit?

Reply:
Note: there are no changes that relates to configuration. The process is simply extract and carry on with the regular flow

Reply:
I'd also like to see `--config-file=cibuildwheel.toml` in a test. We should check that it's resolved relative to the cwd of the invocation, not to the sdist or a temp dir (if that's what's used).